sass/components/_search-form.scss

```diff
@@ -82,3 +82,65 @@ Style guide: siteheader.searchform
     min-height: 2.375em;
   }
 }
+
+/*
+Search form for internal search
+
+Markup: html/search-form-improved.html
+
+.search-form--improved
+
+Style guide: searchform.search-form--improved
+*/
+
+.search-form--improved {
+  background: #f3f3f3;
+  padding: 1.2em 1.5em;
+
+  h5 {
+    font-size: $font-size-medium;
+    text-transform: none;
+    padding-bottom: 1.2em;
+  }
+
+  .search-form__textfield-wrapper {
+    float: none;
+    width: 100%;
+    background: #fff;
+    border: 3px solid #fff;
+    border-bottom: none;
+    @extend .icon;
+
+    &:before {
+      @extend .icon--search;
+      position: absolute;
+      top: 25%;
+      left: 1%;
+      z-index: 3;
+      color: $grey;
+    }
+
+    &:hover {
```

Comment:
I can't see why we should hide the icon on:hover?
Also, if we mouse:out when writing on this field, the icon comes back and sits on top of the copy.

I'd suggest adding this to avoid that:

`.search-form__textfield {
    text-indent: 1.4em;
  }`